### PR TITLE
IWA-922 - Add hotfix github action workflows to shared "Actions" repo

### PR DESCRIPTION
**Pull request by marla294**
https://resideo.atlassian.net/browse/IWA-922

Created `web-create-hotfix-pr` action in "actions" repo to allow the web team to create hotfix PRs automatically using this shared workflow.  We can use this for the Stage and Master hotfix releases on pro-portal, resideo central, and other repos.

Example of how this can be used with external repo such as Pro Portal is shown in the README file, for stage on Pro Portal.

### CODE REVIEW

File: web-create-hotfix-pr/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    types:
+      - closed
+    branches:
+      - develop
+
+jobs:
+  stage-hotfix-rc:
+    name: Stage - Create Hotfix Release Candidate
+    if: |
+      github.event.pull_request.merged == true && (contains(github.head_ref, 'hotfix/') || contains(github.event.pull_request.labels.*.name, 'hotfix') || contains(github.event.pull_request.labels.*.name, 'Hotfix'))
```
Comment (jmurphy-res) [inline web-create-hotfix-pr/README.md:21]: As mentioned in the live review, string checks are not case sensitive according to github's docs.

File: web-create-hotfix-pr/action.yml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  fromBranch:
+    description: 'Source branch name. Ex: develop'
+    required: true
+    type: string
+  toBranch:
+    description: 'Destination branch name. Ex: stage'
+    required: true
+    type: string
+  rcBranchPrefix:
+    description: 'RC branch prefix name. Ex: stage-release'
```
Comment (jmurphy-res) [inline web-create-hotfix-pr/action.yml:13]: Examples can be changed for this hotfix workflow

File: web-create-hotfix-pr/action.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    - name: Get Pull Request Number
+      id: pr-number
+      shell: bash
+      run: echo "pull_request_number=$(gh pr view --json number -q .number || echo "")" >> $GITHUB_OUTPUT
+
+    - name: Comment on Pull Request
+      id: pr-comment
+      shell: bash
+      run: |
+        gh pr comment ${{ steps.pr-number.outputs.pull_request_number }} --body "Please use 'Squash and merge' for this PR"
```
Comment (jmurphy-res) [inline web-create-hotfix-pr/action.yml:82]: Should be merge commit instead

**Review by jmurphy-res — CHANGES_REQUESTED** [review]

File: web-create-hotfix-pr/action.yml (hunk 3 above)
Comment (marla294) [reply web-create-hotfix-pr/action.yml]: Done

File: web-create-hotfix-pr/action.yml (hunk 2 above)
Comment (marla294) [reply web-create-hotfix-pr/action.yml]: Done

File: web-create-hotfix-pr/README.md (hunk 1 above)
Comment (marla294) [reply web-create-hotfix-pr/README.md]: I did a test to double check and what you are saying is true.  I updated the ReadMe to reflect how the logic will look on the Pro Portal action.

**Review by jmurphy-res — APPROVED** [review]

**Pull request merged**